fiftyone/multimodal/tags/_temporal_tags.py

```
@@ -1447,18 +1472,33 @@ def _ensure_indexes(collection) -> None:
     collection.create_index(
         [
             ("_dataset_id", ASCENDING),
+            ("kind", ASCENDING),
             ("anchor", ASCENDING),
             ("tag", ASCENDING),
         ],
         name="temporal_tag_counts",
     )
 
 
+def _ensure_kind(kind):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_ensure_kind`

(ANN202)